.eslintignore

```diff
@@ -1,19 +1,21 @@
-polyfills/_ArrayIterator
-polyfills/Array/of
-polyfills/Array/prototype/values
-polyfills/atob
-polyfills/AudioContext
-polyfills/fetch
-polyfills/Function/prototype/bind
-polyfills/HTMLPictureElement
-polyfills/IntersectionObserver
-polyfills/Intl
-polyfills/JSON
-polyfills/navigator/sendBeacon
-polyfills/Promise
-polyfills/setImmediate
-polyfills/URL
-polyfills/UserTiming
-polyfills/WeakSet
-polyfills/~html5-elements
+polyfills/_ArrayIterator/polyfill.js
+polyfills/Array/of/polyfill.js
+polyfills/Array/prototype/values/polyfill.js
+polyfills/atob/polyfill.js
+polyfills/AudioContext/polyfill.js
+polyfills/fetch/polyfill.js
+polyfills/Function/prototype/bind/polyfill.js
+polyfills/HTMLPictureElement/polyfill.js
+polyfills/IntersectionObserver/polyfill.js
+polyfills/Intl/polyfill.js
+polyfills/Intl/~locale/**/polyfill.js
+polyfills/JSON/polyfill.js
+polyfills/navigator/sendBeacon/polyfill.js
+polyfills/MutationObserver/polyfill.js
+polyfills/Promise/polyfill.js
+polyfills/setImmediate/polyfill.js
+polyfills/URL/polyfill.js
+polyfills/UserTiming/polyfill.js
+polyfills/WeakSet/polyfill.js
+polyfills/~html5-elements/polyfill.js
```

Comment:
This reverses a change I made recently. Sorry if I didn't understand what I was doing. Can you explain why we need to specifically exclude the polyfill.js files rather than the whole generated folder? Is it because you want to lint detects and tests for imported polyfills?